### PR TITLE
fix: 🐛 adding column on XRC20 table #1599 

### LINKED ISSUE #1599: add token name and link in XRC20 tab
image

### PR DESCRIPTION
**Pull request by ZQ-jhon**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
Adding column for XRC20 on table, for fix issue #1599 
**Which issue(s) this PR fixes**:

Fixes #1599 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: src/shared/common/xrc20-token.tsx [hunk 1]
```diff
@@ -127,10 +130,52 @@ const XRC20TokenValue: React.FC<{ contract: string; value: BigNumber }> = ({
...
+    .catch(() => {
+      setTokenAddress("");
+    });
+  if (!validateAddress(tokenAddress)) {
+    return <LinkButton>{tokenAddress}</LinkButton>;
+  }
+  return (
+    <Query
+      query={GET_ADDRESS_META}
+      variables={{ tokenAddress }}
```
Comment (hunglmtb) [inline src/shared/common/xrc20-token.tsx:152]: `GET_ADDRESS_META` requires `address` as variables, could you correct this and post some screenshots test at your local?
```
export const GET_ADDRESS_META = gql`
  query addressMeta($address: String!) {
    addressMeta(address: $address) {
      name
    }
  }
`;
```

**Review by hunglmtb — CHANGES_REQUESTED** [review]

File: src/shared/common/xrc20-token.tsx (hunk 1 above)
Comment (ZQ-jhon) [reply src/shared/common/xrc20-token.tsx]: The component is unused, I have deleted it just now.

File: src/shared/common/xrc20-token.tsx [hunk 2]
```diff
@@ -6,7 +6,6 @@ import { Token } from "../../erc20/token";
 import { TokenNameRenderer } from "../renderer/token-name-renderer";
 import { GetTokenMetadataMap } from "./common-metadata";
 import { numberWithCommas } from "./vertical-table";
-
 const XRC20TokenName: React.FC<{ contract: string }> = ({ contract }) => {
```
Comment (uu-z) [inline src/shared/common/xrc20-token.tsx:9]: please update XRC20TokenName component and add hyperlink support.  it should be ` /token/{tokencontractaddress}`

**Review by uu-z — CHANGES_REQUESTED** [review]

File: src/shared/common/xrc20-token.tsx (hunk 2 above)
Comment (ZQ-jhon) [reply src/shared/common/xrc20-token.tsx:12]: Added it.

**Review by uu-z — APPROVED** [review]

**Pull request merged**